index.html

```diff
@@ -45,22 +47,16 @@ <h2>KEEPERS</h2>
 
     <section class="bgImageWrapper"></section>
     <div class="main">
-        <div class="sidebar">
-            <ul>
-                <li class="sidebar-button">ANIMAL 1</li>
-                <li class="sidebar-button">ANIMAL 2</li>
-                <li class="sidebar-button">ANIMAL 3</li>
-                <li class="sidebar-button">ANIMAL 4</li>
-                <li class="sidebar-button">ANIMAL 5</li>
-                <li class="sidebar-button">ANIMAL 6</li>
-                <li class="sidebar-button">ANIMAL 7</li>
-                <li class="sidebar-button">ANIMAL 8</li>
-                <li class="sidebar-button">ANIMAL 9</li>
-            </ul>
+
+        <div class="sidebar" id="sidebar">  
+            <ul class="animal-list" id="animal-list">
+               
+            </ul>          
         </div>
-        <div class="welcome-message">
-            <h1>WELCOME TO THE "ZOO KEEPERS" OFFICIAL PAGE!</h1>
-            <P>We are thrilled to have you here. Explore and enjoy your time on our platform!</P>
+        <div class="welcome-message" id="main-content">
+            <h1 id="welcome-message" data-default="WELCOME TO THE ZOO KEEPERS OFFICIAL PAGE!">WELCOME TO THE "ZOO KEEPERS" OFFICIAL PAGE!</h1>
```

Comment:
Just saw this.. We can't use h1 for both the logo group name and this one. Please change it to something else and restyle it